cypress/support/step_definitions/index.ts

```diff
@@ -110,12 +145,24 @@ Then('the URL {string} was requested with', (url: string, yaml: string) => {
   })
 })
 
-Then(/^the "(.*)" element( does| doesn't| don't)? exist[s]?$/, function (selector: string, assertion: string) {
-  $(selector).should(`${(assertion || 'does').trim() !== 'does' ? 'not.' : ''}exist`)
+Then(/^the "(.*)" element[s]?( don't | doesn't | do | does | )exist[s]?$/, function (selector: string, assertion: string) {
```

Comment:
I think we said yesterday that we were going to keep this auxiliary verb thing to the minimum of options rather than let folks write this in many different ways.

I think I said yesterday that I probably shouldn't have included the `does` in the first place, and then we said we were going to just keep it at supporting `don't` and `doesn't`, giving us:

```
the "" element exists
the "" elements exist
the "" element doesn't exist
the "" elements don't exist
```

Which covers everything we might want to do with this and it reads nice and natural with the contraction.

Reply:
I’ve removed these two words now.